bot/cogs/defcon.py

```diff
@@ -252,6 +257,22 @@ def mod_log(self) -> ModLog:
                 f"**Days:** {self.days.days}"
             )
 
+        await self.update_channel_topic()
+
+    async def update_channel_topic(self):
+        """
+        Update the #defcon channel topic with the current DEFCON status
+        """
+
+        if self.enabled:
+            day_str = "days" if self.days > 1 else "day"
```

Comment:
Can't use `>` between delta and int. 
```suggestion
             day_str = "days" if self.days.days > 1 else "day"
```